beamerthemegemini.sty

```diff
@@ -128,7 +128,7 @@
     \begin{columns}[c]
     \begin{column}{.2\paperwidth}[T]
```

Comment:
How about rendering on one line?

![screen shot 2018-09-28 at 10 25 51 am](https://user-images.githubusercontent.com/274668/46223765-1dfc5c00-c309-11e8-9500-9faa84b5f3fc.png)

What do you think @ChrisKeefe (cc @ebolyen)?


<details>

```diff
diff --git a/beamerthemegemini.sty b/beamerthemegemini.sty
index 3703d9c..cfdb211 100644
--- a/beamerthemegemini.sty
+++ b/beamerthemegemini.sty
@@ -126,11 +126,11 @@
 {
   \begin{beamercolorbox}{headline}
     \begin{columns}[c]
-    \begin{column}{.2\paperwidth}[T]
-        \hskip5cm
-        \includegraphics[width=0.5\textwidth]{assets/PMI-r}
+    \begin{column}{0.1\paperwidth}
+        \hskip2cm
+        \includegraphics[width=0.8\textwidth]{assets/PMI-r}
     \end{column}
-      \begin{column}{.6\paperwidth}
+      \begin{column}{.8\paperwidth}
         \usebeamerfont{headline}
         \vskip3ex
         \centering
@@ -138,9 +138,8 @@
         {\usebeamerfont{headline author}\usebeamercolor[fg]{headline author}\insertauthor\\[1ex]}
         {\usebeamerfont{headline institute}\usebeamercolor[fg]{headline institute}\insertinstitute\\[1ex]}
       \end{column}
-      \begin{column}{.2\paperwidth}
-          \includegraphics[width=0.4\textwidth]{assets/qiime2}
-          \hskip5cm
+      \begin{column}{.1\paperwidth}
+          \includegraphics[width=0.8\textwidth]{assets/qiime2}
       \end{column}
     \end{columns}
     \vspace{5ex}
diff --git a/poster.tex b/poster.tex
index 80c9f7b..b6d7257 100644
--- a/poster.tex
+++ b/poster.tex
@@ -60,7 +60,7 @@
 % Title
 % ====================

-\title{Enabling Reproducible Microbiome Science through \\ Decentralized Provenance Tracking in QIIME 2}
+\title{Enabling Reproducible Microbiome Science through Decentralized Provenance Tracking in QIIME 2}

 \author{Christopher R. Keefe \and Ahmad Turan Naimey \and J. Gregory Caporaso (advisor)}

```
</details>

Reply:
Thanks for the spelling catch @thermokarst. Fixed. 

I think I'm partial to the blockier look, but either one works. (I like the "white space" around the title block, and think it leads the eye into the visualizations better)

Reply:
I am fine with whatever you decide, I think cutting down some of that whitespace came up in our discussion yesterday.